pkg/azuredisk/nodeserver_test.go

```diff
@@ -131,57 +147,61 @@ func TestGetMaxDataDiskCount(t *testing.T) {
 }
 
 func TestEnsureMountPoint(t *testing.T) {
-	skipIfTestingOnWindows(t)
-	errorTarget := "./error_is_likely_target"
-	alreadyExistTarget := "./false_is_likely_exist_target"
-	azuredisk := "./azure.go"
+	errorTarget, err := testutil.GetWorkDirPath("error_is_likely_target")
+	assert.NoError(t, err)
+	alreadyExistTarget, err := testutil.GetWorkDirPath("false_is_likely_exist_target")
+	assert.NoError(t, err)
+	azuredisk, err := testutil.GetWorkDirPath("azure.go")
+	assert.NoError(t, err)
 
 	tests := []struct {
 		desc        string
 		target      string
-		expectedErr error
+		expectedErr testutil.TestError
 	}{
 		{
-			desc:        "[Error] Mocked by IsLikelyNotMountPoint",
-			target:      errorTarget,
-			expectedErr: fmt.Errorf("fake IsLikelyNotMountPoint: fake error"),
+			desc:   "[Error] Mocked by IsLikelyNotMountPoint",
+			target: errorTarget,
+			expectedErr: testutil.InitNewTestError(
```

Comment:
why would you use a func here? is it more easier to understand if it's a stuct?

Reply:
Initially I had planned on having some separate assertion logic to Darwin, but we might not need that anymore. I will revert back to declaring directly as a struct :)